iphone/Maps/Classes/Common.h

```diff
@@ -20,7 +20,7 @@ static NSString * const kHaveAppleWatch = @"HaveAppleWatch";
 
 static CGFloat const kDefaultAnimationDuration = .2;
 
-static inline NSString * const kApplicationGroupIdentifier()
+static inline NSString * kApplicationGroupIdentifier()
```

Comment:
Константный указатель, а не строка, не играет никакой роли, и компилятор ругается.
Вот если бы NSString const * возвращался - тогда другое дело.

> On 8.8.2016, at 03.00, Vlad Mihaylenko <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/Common.h:
> 
> > @@ -20,7 +20,7 @@ static NSString * const kHaveAppleWatch = @"HaveAppleWatch";
> >  
> >  static CGFloat const kDefaultAnimationDuration = .2;
> >  
> > -static inline NSString * const kApplicationGroupIdentifier()
> > +static inline NSString * kApplicationGroupIdentifier()
> а зачем тут не константный указатель? возвращается ведь из функции NSString * const
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>